src/course/syllabus/developer/projects/DOTNET-intro/resources.md

```diff
@@ -0,0 +1,16 @@
+# Resources
+
+These resources are helpful extra reading that may clarify or enhance concepts you're learning in the main curriculum.
+
+## .NET
+
+
```

Comment:
Typo fixes:
geard -> geared
upe -> up
dotnet -> .NET
I've been been capitalising words like Typescript and Express in the rest of the course book so maybe we keep that convention?